### DISCUSSION TITLE
[@types/chrome] Add new types for chrome.sidePanel

### DISCUSSION THREAD

**Discussion by thdoan** [DefinitelyTyped/DefinitelyTyped#65854] [Issues with a @types package]
Please add support for the new [Side Panel API](https://developer.chrome.com/docs/extensions/reference/sidePanel/) that was introduced in Chrome 114.

[a comment by an automated account is not shown]

**Comment by diamond-fish** [comment]
Related to https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/65814
